trio/tests/test_subprocess.py

```diff
@@ -464,7 +464,7 @@ def broken_terminate(self):
 
```

Comment:
This is no longer testing a warning, so you should change the name of the test (since it's currently "warn_on_failed_cancel_terminate")